frontend/src/pages/Workflows/WorkflowItem.js

```diff
@@ -459,7 +486,9 @@ export const WorkflowItem = SortableElement(
               workflowSortEnabled,
               status,
               () => props.showWorkflowitemAdditionalData(id),
-              additionalData
+              additionalData,
+              props.idsPermissionsUnassigned,
```

Comment:
if you add `idsPermissionsUnassigned` to the destructuring above and in the component call one layer above, you don't need the `props.` (which is favorable, we should avoid putting everything in the props)